docs/source/maintenance/migration.rst

```diff
@@ -0,0 +1,44 @@
+====================
+Migration To RocksDB
+====================
+
+Iroha allows for using Postgres or Rocks Database.
+You can use the option you prefer and if you want to switch -- there is an option for you.
+**Just migrate your database!**
+Here is how
+
+.. hint:: Both migration tool and WSV checker have ``-help`` that you can call to check the available flags anytime.
+
+How to migrate
+==============
+
+After you built Iroha (or simply run it) after version Iroha 1.3 you already have the migration tools as a separate executable! 
+
+Just run the ``irohad/iroha-migrate`` with the following flags:
+
+- ``-help`` - help
+- ``-block_store_path`` - specifies path to block store. Default: "/tmp/block_store"
+- ``-export`` - exports block store to specified directory, default CWD. Use it to *reverse* migration to RocksDB (by exporting files from it).
+- ``-force`` - use it to override blocks in RocksDB blockstore if it already exists. This might be very useful if the next step - checking the correctness of the migrated database - goes through with errors and you need to repeat the migration process. Default: false
+- ``-rocksdb_path`` - specifies the path to RocksDB. Default: "rocks.db"
+
+And your database is migrated! But that is not all.
```

Comment:
Successful output looks like
```
Success! WSV in RocksDB was build.
Next step check consintancy with Postgres WSV using iroha_wsv_diff.
```
> It would be worth to hide example output into <spoiler\>

On error the tool will traditionally exit with non-zero.